lib/dap/filter/geoip2.rb

```diff
@@ -33,6 +33,16 @@ def self.find_db(db_file_names, db_dirs, env_path)
     nil
   end
 
+  def get_maxmind_data(db, ip)
+    geo_hash = nil
+    begin
+      geo_hash = db.get(ip)
+    rescue IPAddr::InvalidAddressError
+    end
+
+    geo_hash
+  end
```

Comment:
```    
begin
    db.get(ip)
rescue IPAddr::InvalidAddressError
end
``` 
might be cleaner.  I don't see a need for the `geo_hash` variable.  Ruby will return nil by default if it enters the rescue block.